samples/react-timeline/src/webparts/timeline/TimelineWebPart.manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/spfx/client-side-web-part-manifest.schema.json",
+  "id": "f6cf705a-2e4b-40d2-9fdd-942883c8f826",
+  "alias": "TimelineWebPart",
+  "componentType": "WebPart",
+
+  // The "*" signifies that the version should be taken from the package.json
+  "version": "*",
+  "manifestVersion": 2,
+
+  // If true, the component can only be installed on sites where Custom Script is allowed.
+  // Components that allow authors to embed arbitrary script code should set this to true.
+  // https://support.office.com/en-us/article/Turn-scripting-capabilities-on-or-off-1f2c515f-5d7e-448a-9fd7-835da935584f
+  "requiresCustomScript": false,
+  "supportedHosts": ["SharePointWebPart"],
+
+  "preconfiguredEntries": [{
+    "groupId": "5c03119e-3074-46fd-976b-c60198311f70", // Other
+    "group": { "default": "Other" },
+    "title": { "default": "Timeline" },
+    "description": { "default": "Displays a list of events in chronological order." },
+    "officeFabricIconFontName": "TimelineProgress",
+    "properties": {
+      "description": "Timeline Events",
+      "listName": "Timeline",
+      "layout": "Vertical",
+      "showImage": true,
+      "showDescription": true,
+      "dateFormat": "MM/DD/yyyy",
```

Comment:
If you localize, you could possibly leverage the concepts used at https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/guidance/localize-web-parts to localize your date format. That way, the default date format will appear for users in every language.